src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss

```diff
@@ -31,6 +31,7 @@ div {
   text-transform: uppercase;
   color: var(--primary-green);
   margin-right: 5px;
+  display: inline-block;
```

Comment:
use class d-inline instead
https://getbootstrap.com/docs/4.0/utilities/display/

Reply:
ok for class. but d-inline-block. It must be height as block element and alignment as inline element.

Reply:
done